packaging/builder-images/Dockerfile.x86_64-unknown-linux-musl

```diff
@@ -16,21 +16,33 @@ RUN temp_dir=$(mktemp -d) \
   && cd $temp_dir \
   && curl -L https://github.com/richfelker/musl-cross-make/archive/v0.9.9.tar.gz | tar --strip-components=1 -xz \
   && make install -j$(nproc) \
-    GCC_VER=9.2.0 \
-    MUSL_VER=1.2.0 \
-    DL_CMD="curl -C - -L -o" \
-    OUTPUT=/usr/local/ \
-    TARGET=x86_64-linux-musl \
+  GCC_VER=9.2.0 \
+  MUSL_VER=1.2.0 \
+  DL_CMD="curl -C - -L -o" \
+  OUTPUT=/usr/local/ \
+  TARGET=x86_64-linux-musl \
   && rm -rf $temp_dir
 
-ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER=x86_64-linux-musl-gcc \
-    CC_x86_64_unknown_linux_musl=x86_64-linux-musl-gcc \
-    CXX_x86_64_unknown_linux_musl=x86_64-linux-musl-g++
 
-RUN rustup target add x86_64-unknown-linux-musl
+RUN apt-get update; \
+  apt-get install -y \
+  cmake        `# for building C deps` \
+  libclang-dev `# for onig_sys (via the regex crate)` \
+  libssl-dev   `# for openssl (via surf)` \
+  libsasl2-dev `# for librdkafka` \
+  libzstd-dev  `# for librdkafka`; \
+  apt-get autoremove -y \
+  && apt-get clean -y \
+  && rm -rf /var/lib/apt/lists/*
+
+#COPY shared/entrypoint.sh /
+#ENTRYPOINT [ "/entrypoint.sh" ]
+
+
+ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER=x86_64-linux-musl-gcc \
+  CC_x86_64_unknown_linux_musl=x86_64-linux-musl-gcc \
+  CXX_x86_64_unknown_linux_musl=x86_64-linux-musl-g++
 
-COPY shared/install_dependencies_debian.sh /
-RUN /install_dependencies_debian.sh
 
-COPY shared/entrypoint.sh /
-ENTRYPOINT [ "/entrypoint.sh" ]
+# COPY shared/entrypoint.sh /
```

Comment:
commented out on purpose?

Reply:
Ja I didn't wanted to loose those lines, right now musl is blocked on the snmalloc issue: https://github.com/SchrodingerZhu/snmalloc-rs/issues/102

the Dockerfile isn't working as it is right now and it's unclear what is needed to get it working once the snmalloc issue is resolved so I wanted to keep the 'old' docker bits in there as a comment in case it turns later out we needed them after all.